spec/db/chores/roles_conversion_spec.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../../../db/chores/roles_conversion"
```

Comment:
this seems brittle. what happens when we (inevitably) reorganize the file structure of the tests?